front/lib/conversation.ts

```diff
@@ -0,0 +1,131 @@
+import {
```

Comment:
Not sure if this is the best place for this.... maybe front/components/lib? to make it clear it's a client-side thing. We def don't want a gazilion files in front/lib IMHO?

Reply:
@app/components/assistant/conversation/lib.ts ?